### DISCUSSION TITLE
Add 365inspect controls to maester

### DISCUSSION THREAD

**Discussion by softengsistemas** [maester365/maester#752] [Ideas]
Hi!

I think that it's a good idea to add some 365 inspectors from 365inspect in maester, in order to increase the capabilities of the maester detections to Exchange, Sharepoint, OneDrive and Teams. Attached the github repository of 365 inspect (https://github.com/soteria-security/365Inspect) and the current inspectors (https://github.com/soteria-security/365Inspect/tree/main/Inspectors).

**Comment by soulemike** [comment]
Those look like they are some good tests. The project is listed as under MIT, but if you don't mind trying to see if a maintainer has any concerns with incorporating the tests here as well.

It would likely be best to use a process similar to EIDSCA tests or what is staged for ORCA. That way the tests can be generated dynamically as the upstream module changes.
